### PR TITLE
[LLVM][SYCL][E2E] Disable xfail for passing test

### PR DESCRIPTION
**Pull request by ggojska**
Disabled xfail in following tests:
    
    SYCL :: Matrix/SG32/get_coord_int8_matB.cpp
    SYCL :: Matrix/get_coord_int8_matB.cpp
    SYCL :: Matrix/joint_matrix_annotated_ptr.cpp

### CODE REVIEW

File: sycl/test-e2e/Matrix/get_coord_int8_matB.cpp [hunk 1]
```diff
@@ -9,7 +9,6 @@
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
-// XFAIL: *
```
Comment (dkhaldi) [inline sycl/test-e2e/Matrix/get_coord_int8_matB.cpp:12]: @Nuullll, can you please comment on whether we should keep xfail::cpu here?
"get_coord" was also corrected on the CPU for packed B, right?

File: sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp [hunk 2]
```diff
@@ -11,8 +11,6 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// XFAIL: *
```
Comment (dkhaldi) [inline sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp:14]: Same here:
@Nuullll, can you please comment on whether we should keep xfail::cpu here?
"get_coord" was also corrected on the CPU for packed B, right?

**Comment by dkhaldi** [comment]
@ggojska, you should create a fork from the repo first.

File: sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp (hunk 2 above)
Comment (Nuullll) [reply sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp:14]: "XFAIL: cpu" should be kept here.

File: sycl/test-e2e/Matrix/get_coord_int8_matB.cpp (hunk 1 above)
Comment (Nuullll) [reply sycl/test-e2e/Matrix/get_coord_int8_matB.cpp:12]: "XFAIL: cpu" should be kept here.

**Pull request closed without merging**